petab/lint.py

```diff
@@ -166,6 +166,23 @@ def check_parameter_df(
             assert_no_leading_trailing_whitespace(
                 df[column_name].values, column_name)
 
+    # nominal value is generally optional, but required if any for any
+    #  parameter estimate != 1
+    non_estimated_par_ids = list(df.index[(df[ESTIMATE] != 1)
+                                          | (df[ESTIMATE] != '1')])
+    if non_estimated_par_ids:
+        if NOMINAL_VALUE not in df:
+            raise AssertionError("Parameter table contains parameters "
+                                 f"{non_estimated_par_ids} that are not "
+                                 "specified to be estimated, "
+                                 f"but column {NOMINAL_VALUE} is missing.")
+        try:
+            df.loc[non_estimated_par_ids, NOMINAL_VALUE].apply(float)
```

Comment:
```suggestion
            df.loc[non_estimated_par_ids, NOMINAL_VALUE].apply(int)
```

Reply:
ist ``in [0,1]`` explicitly checked anywhere?

Reply:
yes `assert_parameter_estimate_is_boolean`

Reply:
But this one is about nominalValue, not about estimate. So float seems more appropriate.